hoodie-client/src/main/java/com/uber/hoodie/table/HoodieCopyOnWriteTable.java

```diff
@@ -414,6 +417,12 @@ public boolean isWorkloadProfileNeeded() {
     return true;
   }
 
+  @Override
+  public Iterator<List<WriteStatus>> handleCompactionPartition(Iterator<CompactionOperation> operation,
```

Comment:
nit: rename to `handleCompaction` to be consistent with other methods

Reply:
So the other methods are handleUpsertPartition, handleInsertPartition..So this is consistent with those. The handleCompaction(..) method is added to HoodieMergeOnReadTable just like handleUpsert/handleInsert in HoodieCopyOnWriteTable..

Reply:
if its consistent all good..